modules/videoio/src/cap_v4l.cpp

```diff
@@ -1803,6 +1803,9 @@ bool CvCaptureCAM_V4L::setProperty( int property_id, double _value )
         if (bool(value)) {
             convert_rgb = convertableToRgb();
             return convert_rgb;
+        }else{
+            convert_rgb = false;
+            releaseFrame();
         }
         convert_rgb = false;
```

Comment:
Lets keep code cleaner:
- remove duplicated `convert_rgb = false;`
- remove "else" body or move return into this "else" body.

Reply:
Done and done.